Core/Orbwalker.cs

```diff
@@ -251,6 +263,16 @@ public static AttackableUnit GetTarget(OrbwalkerMode? modeArg)
                         return minion;
                     }
                 }
+                foreach (var minion in
+                    GameObjects.AttackableUnits.Where(m => m.IsValidTarget(m.GetRealAutoAttackRange()))
+                        .Select(barrel => barrel as Obj_AI_Minion)
+                        .Where(
+                            m =>
+                            m != null && m.CharData.BaseSkinName == "gangplankbarrel" && m.IsHPBarRendered
+                            && m.Health < 2))
```

Comment:
so it won't check for 0 because it should never be null. I only think that it would be better to be precisely.